### PR TITLE
Austria-Oesterreichische-Nationalbank-OENB-Sanctions-#211

### PR DESCRIPTION
**Pull request by nvmbrasserie**

### CODE REVIEW

**Review by jbothma — COMMENTED** [review]
ugh, looks like I never sent my comments, sorry

File: datasets/au/oenb_sanctions/au_oenb_sanctions.yml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+publisher:
+  name: Austrian National Bank
+  acronym: OeNB
+  description: |
+    The Oesterreichische Nationalbank (OeNB) is the central bank of the Republic of Austria and as such an integral part of the European System of Central 
+    Banks (ESCB). The OeNB contributes, in the public interest, to monetary and economic policy decision-making in Austria and the euro area. Its core business 
+    areas are cash management, monetary policy, financial stability, statistics, and payments.
+    [Source: Official Website](hhttps://www.oenb.at/en/)
+  url: https://www.oenb.at
+  official: false
```
Comment (jbothma) [inline datasets/au/oenb_sanctions/au_oenb_sanctions.yml:26]: this is official - we've done the data extraction from official documents, right?

**Comment by pudo** [comment]
One very dumb catch: au is Australia, at is Austria :)

File: datasets/au/oenb_sanctions/au_oenb_sanctions.yml [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+deploy:
+  schedule: "@weekly"
+load_db_uri: ${OPENSANCTIONS_DATABASE_URI}
+summary: >
+  This dataset compiles Austrian National Bank regulations aimed at combating terrorism through restrictions on capital and payment transactions, 
+  covering the assets of designated individuals under the 2010 Sanctions Act.
+description: |
+  The dataset comprises unofficial versions of regulations provided by the Austrian National Bank (OeNB) aimed at combating terrorism through capital
+  and payment traffic restriction measures. It covers regulations DL 2/2002 and its 8th amendment DL 1/2009. These regulations are considered valid under 
+  the 2010 Sanctions Act and cover assets of specified organizations and individuals.
```
[the remark this answers is not in the document]
Comment (nvmbrasserie) [reply datasets/au/oenb_sanctions/au_oenb_sanctions.yml:16]: That's what they claim on their website: 

'Regulations according to DevG

The texts offered are non-official versions. The OeNB makes every effort to present the contents of the regulations listed below in their current version or to refer to the current version. However, the OeNB assumes no liability for any deviations that nevertheless occur.'

That's why I decided to frame it this way.

File: datasets/at/nbter_sanctions/crawler.py [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+    birth_place = row.pop("place of birth")
+    id_number = row.pop("ID card no.")
+    source = row.pop("source")
+    entity_type = row.pop("type")
+
+    entity = None
+    if entity_type == "Person":
+        entity = context.make("Person")
+        entity.id = context.make_id(full_name, birth_place, birth_date)
+        entity.add("name", full_name)  # still need to split it
```
Comment (jbothma) [inline datasets/at/nbter_sanctions/crawler.py:53]: what splitting is still needed? no need to split on comma and change the order.
if there's nothing more to do, you can delete the comment.

File: datasets/at/nbter_sanctions/at_nbter_sanctions.yml [hunk 4]
```diff
@@ -0,0 +1,40 @@
+title: Austrian National Bank Regulations on Terrorism Financing Restrictions
+entry_point: crawler.py
+prefix: at-nbter
+coverage:
+  frequency: weekly
```
Comment (jbothma) [inline datasets/at/nbter_sanctions/at_nbter_sanctions.yml:5]:
```suggestion
  frequency: never
```

**Pull request merged**